xbmc/utils/FileUtils.cpp

```diff
@@ -201,3 +203,59 @@ CDateTime CFileUtils::GetModificationDate(const std::string& strFileNameAndPath,
   }
   return dateAdded;
 }
+
+bool CFileUtils::CheckFileAccessAllowed(const std::string &filePath)
+{
+  // DENY access to paths matching
+  const std::vector<std::string> blacklist = {
+    "passwords.xml",
+    "sources.xml",
+    "guisettings.xml",
+    "advancedsettings.xml",
+    ".ssh",
+    "/etc"
```

Comment:
Surely "/etc" could be found in valid folder paths too

Reply:
debateable. If you can show an example, I'll remove it

Reply:
remember this is a find, not a startswith wsnipex

/home/etcetera/

Reply:
/etc/ better?

Reply:
Yeap, and "etc" itself is the name of several artists so could easily be a folder name

Reply:
Why do we have to put `/etc` separately on the blacklist? Isn't it supposed to be blocked anyway due to not being under any of the whitelisted locations?